charts/scalardb/values.yaml

```diff
@@ -0,0 +1,109 @@
+# Default values for scalardb.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# nameOverride -- String to partially override scalardb.fullname template (will maintain the release name)
+nameOverride: ""
+# fullnameOverride -- String to fully override scalardb.fullname template
+fullnameOverride: ""
+
+scalardb:
+  # replicaCount -- Default values for number of replicas.
+  replicaCount: 3
+
+  # scalardb.contactPoints -- The database contanct point such as a hostname of Cassandra or a URL of Cosmos DB account.
+  contactPoints: cassandra
+  # scalardb.contactPort -- The database port number.
+  contactPort: 9042
+  # scalardb.username -- The username of the database. For Cosmos DB please leave blank. For Dynamo DB please specify key id here.
+  username: cassandra
+  # scalardb.password -- The password of the database. For Cosmos DB, Dynamo DB please specify a secret key here.
+  password: cassandra
+  # scalardb.storage -- Storage implementation. Either cassandra or cosmos or dynamo or jdbc can be set.
+  storage: cassandra
+
+  server:
+    # scalardb.server.port -- The port of Scalar DB server.
+    port: 60051
+    # scalardb.server.prometheus_http_endpoint_port -- The port of Prometheus service runnning in Scalar DB server.
+    prometheus_http_endpoint_port: 8080
```

Comment:
Do we need to customize these ports? Since these are inside a container, I think we can use default ports.

Reply:
If we do that then we have to set these ports as default in the generated config file in the `config-map.yaml` as well. 
Should we do that or still keep it customizable?

Reply:
I don't think we need to set these ports in `config-map.yaml`, Because I think the following source will use the default ports, am I wrong?
https://github.com/scalar-labs/scalardb/blob/1e3e69a8c28f03e52854273bab9433355947dad4/server/src/main/java/com/scalar/db/server/config/ServerConfig.java#L24-L25

Reply:
Ah yes. So we will remove them from `config-map.yaml` and using the default one then?

Reply:
Yes! I think so!

Reply:
Thank you!